litmus-portal/frontend/src/components/Sections/Settings/AccountsTab/PersonalDetails/index.tsx

```diff
@@ -16,10 +18,16 @@ interface personaData {
 // Displays the personals details on the "accounts" tab
 const PersonalDetails: React.FC = () => {
   const classes = useStyles();
-
   const userData: UserData = useSelector((state: RootState) => state.userData);
+  const { username } = userData;
+
+  // Query to get user details
+  const { data } = useQuery<CurrentUserDetails>(GET_USER, {
+    variables: { username },
+  });
 
-  const { name, email, username } = userData;
+  const name: string = data?.getUser.name as string;
```

Comment:
this typecasting also shouldn't be required...if the interfaces are written correctly